Dockerfile.dev

```diff
@@ -1,21 +1,13 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.9
```

Comment:
Let’s keeping use the official devcontainers.

Reply:
The official devcontainer is nothing but the official Python + common_utils + pipx tools (which currently are being uninstalled in the Dockerfile -- like the pipx tools).

This way we avoid having to download an unnecessary layer with tools we will uninstall later.

common_utils is being added back as a feature in the devcontainer.json.

Reply:
Still I think we should use devcontainers.

Reply:
Ok, I still think it's a mistake but I reverted back, to the new repository btw.